osfoffline/client/osf.py

```diff
@@ -59,8 +67,21 @@ def from_data(cls, request_session, data):
     @classmethod
     def load(cls, request_session, *args, **kwargs):
         resp = request_session.get(cls.get_url(*args, **kwargs), params={'page[size]': 250})
+        if (resp.status_code % 500) <= 100:
```

Comment:
For clarity, can you please use `if resp.status_code >= 500`?

(unless there is a benefit to using modulo here...?)